wire-ios-sync-engine/Tests/Source/Calling/WireCallCenterV3Tests.swift

```diff
@@ -895,6 +895,8 @@ class WireCallCenterV3Tests: MessagingTest {
         // and when the conference info changes
         conferenceInfoChangeSubject.send(conferenceInfo2)
 
+        XCTAssert(waitForAllGroupsToBeEmpty(withTimeout: 0.5))
```

Comment:
@typfel  we added several waiting statement and it fixes the tests.
I wonder, however, if the execution order will be right in production.

Reply:
I think it's fine it's mostly about removing stale participants from a call and the order is not critical there.